infrastructure/scripts/execution-plan/container.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+set -ueo pipefail
+echo "eu.gcr.io/freiheit-core/images/execution-plan:2.0-scratch-NG-8"
```

Comment:
Looks good to me, but why you’re putting the container name ina separate script?

Reply:
refactored code to have common code, so that if container is updated in one place, it gets updated in other script as well.